src/StelSplashScreen.hpp

```diff
@@ -27,16 +27,10 @@ class SplashScreen
 {
 	class SplashScreenWidget : public QSplashScreen
 	{
+        QFont splashFont;
```

Comment:
Please fix indentation. (use the code style file!)

Reply:
Always forget to switch editor settings :/
Fixed now.